packages/node_modules/@webex/internal-plugin-devices/src/features/feature-model.js

```diff
@@ -0,0 +1,139 @@
+// External dependencies.
+import AmpState from 'ampersand-state';
+import {defaults, isObject} from 'lodash';
+
+/**
+ * Feature model.
+ *
+ * @description
+ * This model contains details on a single feature and is received from the
+ * **WDM** service upon registration.
+ */
+const FeatureModel = AmpState.extend({
+
+  // Ampersand property members.
+
+  props: {
+    /**
+     * Contains the unique identifier for this feature to be addressed by.
+     *
+     * @type {string}
+     */
+    key: 'string',
+
+    /**
+     * This property contains the date in which this feature was last modified.
+     *
+     * @type {date}
+     */
+    lastModified: 'date',
+
+    /**
+     * This property defines whether or not the feature is mutable.
+     *
+     * @type {boolean}
+     */
+    mutable: 'boolean',
+
+    /**
+     * This property contains the data type the string value should be
+     * interpreted as.
+     *
+     * @type {string}
+     */
+    type: 'string',
+
+    /**
+     * This property contains the string value of this feature.
+     *
+     * @type {string}
+     */
+    val: 'string',
+
+    /**
+     * This property contains the interpreted value of this feature.
+     *
+     * @type {any}
+     */
+    value: 'any'
+  },
+
+  /**
+   * Class object constructor. This method safely initializes the class object
+   * prior to it fully loading to allow data to be accessed and modified
+   * immediately after construction instead of initialization.
+   *
+   * @override
+   * @param {any} attrs
+   * @param {any} options
```

Comment:
Should this be an object? Also, can you document the default value?

Reply:
Updated the documentation.